.github/workflows/pylint.yml

```diff
@@ -0,0 +1,44 @@
+name: Linter
```

Comment:
All workflows should be named differently.

Reply:
Not obligatory. There are no tools used to work with workflow through the name. The display of the job name in the checkers field will also depend on this.

Reply:
OK, fair enough.